sunpy/map/mapbase.py

```diff
@@ -2174,11 +2174,28 @@ def draw_contours(self, levels, axes=None, **contour_args):
         axes = self._check_axes(axes)
 
         levels = self._process_levels_arg(levels)
-        kwargs = {}
-        if wcsaxes_compat.is_wcsaxes(axes):
-            kwargs['transform'] = axes.get_transform(self.wcs)
-        kwargs.update(contour_args)
-        cs = axes.contour(self.data, levels, **kwargs)
+
+        # Pixel indices
+        x, y = np.meshgrid(*map(np.arange, np.flipud(self.data.shape)))
```

Comment:
Heh, this craziness is unnecessary.  I replaced it with `y, x = np.indices(self.data.shape)`.

As a matter of course, I'd prefer to avoid having `mapbase.py` code depend on `maputils.py`, but given my simplification, it's not even worth doing.  The `maputils.py` uses of `meshgrid()` can also be replaced with `indices()`.

Reply:
> The `maputils.py` uses of `meshgrid()` can also be replaced with `indices()`.

Done in #5419